Makefile

```
@@ -1,32 +1,50 @@
 HTMLDIR=public
-USERNAME=rgommers
 
 BASEURL ?=
 
 ifdef BASEURL
         BASEURLARG=-b $(BASEURL)
 endif
 
-.PHONY: help clean html
+.PHONY: help clean html serve serve-dev teams teams-clean
 
 
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
-	@echo "  upload USERNAME=user   to upload the generated pages to scipy.org"
 	@echo "  html                   to make standalone HTML files"
+	@echo "  serve                  to serve the website locally"
+	@echo "  serve-dev              to serve the website locally and full rebuilds one change"
+	@echo "  teams                  to generate teams files"
 
 clean: ## remove the build artifacts, mainly the "public" directory
 	rm -rf $(HTMLDIR)
 
 prepare: clean
 	git submodule update --init --recursive
 
+TEAMS_DIR = static/teams
+TEAMS = scipy-core-team triage # website
```

Originally excluded to not delete the website file because we don't yet have the team, but I will add it to not forget.
```suggestion
TEAMS = scipy-core-team triage website
```